### PR TITLE
Vampirika/balance warsong

### PR DESCRIPTION
**Pull request by vampirika**
### General

- Fixed legion units getting blocked in Alterac.

### Warsong

- Blademaster shorter and more impactful bladestorm
  - Damage increased 90 -> 160
  - Duration decreased 2s -> 0.75s
  - Total damage now 120, down from 180

- Raiders slightly faster, more hp, given high health regen to fit "raider" theme.
  - Raider health increased 875 -> 925
  - Raider health regen 0.25 -> 2.5
  - Raider movespeed 270 -> 280
  - Chaos Raider health increased 875 -> 1125
  - Chaos Raider health regen 0.25 -> 3.0
  - Chaos Raider movespeed 270 -> 280

- Fixed some chaos counterparts not having enough health.
  - Chaos blademaster 1025 -> 1275 (blademaster 1075 hp)
  - Chaos peon 250 -> 450 (peon 250 hp)
  - Chaos raider (see above)

- Fixed tooltip for Varok battleshout (and indirectly, falstad)

### Fel Horde

- Raiders slightly faster, more hp, given high health regen to fit "raider" theme.

  - Fel Raider health increased 875 -> 925
  - Fel Raider health regen 0.25 -> 2.5
  - Fel Raider movespeed 270 -> 280

### Goblin

- Reduced frequency of ICBM nuke.
  - Building max mana decreased 32000 -> 28000
  - Cast cost increased 4000 -> 7000
  - Cooldown increased 25s -> 45s

### CODE REVIEW

File: mapdata/WarcraftLegacies/AbilityData/Skin/1162424385.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
     {
       "Id": 1953854049,
       "Type": 3,
-      "Value": "The hero lets loose an inspiring shout that strengthens nearby troops for 20 seconds.|n|n|cffffcc00Level 1|r - 2 bonus armor, 200% regen|n|cffffcc00Level 2|r - 3 bonus armor, 300% regen|n|cffffcc00Level 3|r - 4 bonus armor, 400% regen|n|cffffcc00Level 4|r - 5 bonus armor, 500% regen"
+      "Value": "The hero lets loose an inspiring shout that strengthens nearby troops.|n|n|cffffcc00Level 1|r - \u003CA0IE,DataB1\u003E bonus armor, \u003CA0IE,DataC1\u003E bonus regen|n|cffffcc00Level 2|r - \u003CA0IE,DataB2\u003E bonus armor, \u003CA0IE,DataC2\u003E bonus regen|n|cffffcc00Level 3|r - \u003CA0IE,DataB3\u003E bonus armor, \u003CA0IE,DataC3\u003E bonus regen|n|cffffcc00Level 4|r - \u003CA0IE,DataB4\u003E bonus armor, \u003CA0IE,DataC4\u003E bonus regen"
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/AbilityData/Skin/1162424385.json:49]: Good stuff!!

File: mapdata/WarcraftLegacies/UnitData/Core/nchr.json [hunk 2]
```diff
@@ -72,15 +72,15 @@
     {
       "Id": 1919969397,
       "Type": 2,
-      "Value": 0.25
+      "Value": 2.5
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/UnitData/Core/nchr.json:75]: This is hit point regeneration, right?
Legacies units have remarkably consistent hit point regeneration rates so an outlier like this should be indicated somehow. I'd just change the Raider core tooltip to be something like "mobile siege unit **with high hit point regeneration.**" (Replacing the non-buld part with whatever it is now).

File: mapdata/WarcraftLegacies/AbilityData/Skin/1162424385.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
     {
       "Id": 1718840929,
       "Type": 3,
-      "Value": "(Yellow - Falstad)"
+      "Value": "(Yellow - Falstad, Warsong - Varok)"
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/AbilityData/Skin/1162424385.json:15]:
```suggestion
      "Value": "(Falstad, Varok)"
```
Pretty trivial but I think we should just stop putting faction names and colors in ability names when we're already putting the unit in. It's duplicate info.
(Good to add Varok in here though for sure)

File: mapdata/WarcraftLegacies/UnitData/Core/nchr.json (hunk 2 above)
Comment (vampirika) [reply mapdata/WarcraftLegacies/UnitData/Core/nchr.json:75]: Added in

File: mapdata/WarcraftLegacies/AbilityData/Skin/1162424385.json (hunk 3 above)
Comment (vampirika) [reply mapdata/WarcraftLegacies/AbilityData/Skin/1162424385.json:15]: Yup makes sense

**Comment by vampirika** [comment]
I plan on doing more changes, but will do them in another PR to keep stuff bite sized

**Review by YakaryBovine — APPROVED** [review]
Looks good to me.
I have not tested changes in game - changes appear to be numerical and I can see the associated tooltip changes in GitHub (or those tooltips use data fields anyway and thus don't need to be changed).

**Pull request merged**